src/axom/sina/CMakeLists.txt

```diff
@@ -45,15 +45,30 @@ set(sina_sources
     core/Run.cpp
     )
 
+# Set CMake Policy
+cmake_policy(SET CMP0115 NEW)
+
 # Add Adiak header and source
 blt_list_append( TO sina_headers ELEMENTS core/AdiakWriter.hpp IF AXOM_USE_ADIAK )
 blt_list_append( TO sina_sources ELEMENTS core/AdiakWriter.cpp IF AXOM_USE_ADIAK )
 
 # Add fortran interface for Sina
+if(AXOM_USE_HDF5 AND ENABLE_FORTRAN)
+  set(AXOM_USE_HDF5_FORTRAN ".true.")
+else()
+  set(AXOM_USE_HDF5_FORTRAN ".false.")
+endif()
+
 if (ENABLE_FORTRAN)
-  blt_list_append( TO sina_headers ELEMENTS interface/sina_fortran_interface.h)
-  blt_list_append( TO sina_sources
-                   ELEMENTS interface/sina_fortran_interface.cpp interface/sina_fortran_interface.f)
+  blt_list_append(TO sina_headers 
+    ELEMENTS interface/sina_fortran_interface.h)
+  blt_list_append(TO sina_sources 
+    ELEMENTS interface/sina_fortran_interface.cpp interface/sina_fortran_interface.f)
+  configure_file(interface/sina_fortran_interface.f.in 
+                 interface/sina_fortran_interface.f 
+                 @ONLY)
+  install(FILES ${CMAKE_CURRENT_BINARY_DIR}/interface/sina_fortran_interface.f
```

Comment:
Are you sure this file needs to be installed? I could be wrong but I don't think there is a reason to install Fortran source files, just module files. We don't have any installed currently. @ltaylor16 could you weigh in given this isn't my strong suit?

Reply:
correct, only the compiled module files need to be installed.